sources/packages/api/src/constants/error-code.constants.ts

```diff
@@ -53,3 +53,17 @@ export const EDUCATION_PROGRAM_NOT_FOUND = "EDUCATION_PROGRAM_NOT_FOUND";
 
 export const STUDENT_ACCOUNT_APPLICATION_NOT_FOUND =
   "STUDENT_ACCOUNT_APPLICATION_NOT_FOUND";
+
+/**
+ * While creating the student account, the same SIN was already associated
+ * with other students.
+ */
+export const STUDENT_ACCOUNT_CREATION_MULTIPLES_SIN_FOUND =
```

Comment:
IMO, the constant can be renamed to STUDENT_ACCOUNT_CREATION_SIN_ALREADY_ASSOCIATED. That goes better with the comments. Just a suggestion.

Reply:
As per the discussion, I am okay with the naming